### ISSUE TITLE
Collapse / expand button disappear with module activate on WFRP 4e

### ISSUE THREAD

**Issue by alexremy** [ironmonk88/monks-little-details#55]
When module is activated, and if the sidebar is collapsed, the button to re-expand has disappeared. I desactivated all modules except Monk's Little Details and had 100% reproduction of the issue. When module is uninstalled, the issue disappears.

**Comment by ironmonk88** [comment]
Did that just start?  Or has that been a thing for awhile.
Little Details shifts the button up to the top, so if you accidentally collapse the sidebar you don't have to move your mouse down to expand it again, the button is right where the collapse button was.
It looks like warhammer is doing some really weird styling on those buttons, that doesn't entirely make sense.  And I was wondering if the developers of the system introduced an issue.
I've corrected the code on my end so that warhammer should appear correctly, but if the developers didn't intend their changes and fix it, the button will disappear again.

Fix should appear in the next release.

**Comment by alexremy** [comment]
Thanks for getting back to me this quick, really appreciated! It started weeks ago, and I'm assuming it was right when I installed Monk's the first time, but I did noticed yet. I kept living with the issue as refreshing page make the sidebar re-expand by default.
I'm attaching two screenshots (collapsed & expanded) to show you what I am getting in terms of display. I took them right this minute with Monk's installed (and same again, the issue is not here when the module is uninstalled). 

![collapsed](https://user-images.githubusercontent.com/84489957/118985275-5040a900-b94c-11eb-9f2b-b6fee1b2dcc6.png)
![expanded](https://user-images.githubusercontent.com/84489957/118985299-559df380-b94c-11eb-9865-f2ca6a29ce4b.png)
